src/Name/Generator/ControllerNameGenerator.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
+
+namespace Jaeger\Symfony\Name\Generator;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\Event\TerminateEvent;
+
+class ControllerNameGenerator implements NameGeneratorInterface, EventSubscriberInterface
+{
+    private string $controller;
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            RequestEvent::class => ['onRequest', 31],
```

Comment:
If you know the reason, why this priority number used - please specify it here in comment